LDAR_Sim/simulations/M_satellite_dev.yaml

```diff
@@ -0,0 +1,13 @@
+parameter_level: method
+version: '2.0'
+label: satellite_dev
+module: dummy
+deployment_type: orbit
+measurement_scale: site
+sensor: default
+is_follow_up: False
+MDL: [278]
```

Comment:
I don't think we need MDL for satellites (not only GHGSAT), since most satellites' MDL is not a constant. It is a proportional function of the wind speed.